integration_test/agents/agents.go

```diff
@@ -664,6 +666,115 @@ func InstallStandaloneWindowsMonitoringAgent(ctx context.Context, logger *log.Lo
 	return err
 }
 
+// PackageLocation describes a location where packages
+// (currently, only the Ops Agent packages) live.
+type PackageLocation struct {
```

Comment:
this is a lot of inferred behavior, can we make the struct be more strict perhaps? do do validation in it's creation?

Reply:
PTAL, I added code to throw an error for conflicting install locations, not sure if that is sufficient. We could throw an error when creating the struct in the first place, but I think that wouldn't be any better than throwing an error when we try to use the struct (which is what I did) because that happens immediately afterr creating the struct anyways. In other words, the error would be returned at the same moment anyway, it would just cost a slight amount of additional code at each callsite for PackageLocationForEnvVars.